samples/drivers/audio/i2s_codec/README.rst

```diff
@@ -0,0 +1,33 @@
+.. zephyr:code-sample:: i2s-codec
+   :name: Audio Codec (codec)
+   :relevant-api: audio_codec_interface
+
+   Demo Audio Codec with I2s output.
+
+Overview
+********
+
+This is a very simple application intended to show how to use codec with i2s.
+it is read from the i2s buffer and sent to the CODEC.
+the audio data will be output to Lineout of CODEC.
+
+uart needs to be direct to flexcomm12 thourhg arduio uart,
+as flexcomm0 is used for PCM transfer from DMIC.
```

Comment:
if it's a generic I2S sample it probably shouldn't refer to NXP specific IP?

Reply:
let me try to make it platform independ